### DISCUSSION TITLE
External link preview and/or link-type note

### DISCUSSION THREAD

**Discussion by kjgcoop** [TriliumNext/Notes#177] [Ideas]
I posted this over in the original [Trilium discussion boards](https://github.com/zadam/trilium/discussions/4802#discussioncomment-9766664) and was directed over here. Sorry if it's noise rather than signal.

Is there any notion to add functionality to fetch page information when an external URL is added? Like I paste a URL and Trilium retreives its human name and maybe a thumbnail? Even if it was a new note type such that each note could only have the one link.

Below is what it looks like in Linkwarden:
![Screenshot_20240611_141129_Firefox](https://github.com/TriliumNext/Notes/assets/51257881/0395311d-8375-419e-ab90-517ecd0e649c)

I realize this is nontrivial and far from vital, I just think it might be cool. It would be nice keep all my notes in Trilium/Trilium Next (I just learned of this project, so I haven't switched yet - I'm planning to).

Is there any chance to may come to be?

Thank you!

**Comment by CobriMediaJulien** [comment]
We had a dev meeting yesterday and we want to focus at first on porting the code base to typeskript, test this and then developing a proper plugIn system. So we can let this open for the backlock, but right it´s not planned for the next release/ releases.

Greetings Julien

**Reply by kjgcoop** [reply]
Awesome, thanks for letting me know!
